### ISSUE TITLE
new version break nodebb

### ISSUE THREAD

**Issue by sohrabafard** [NodeBB-Community/nodebb-plugin-recent-cards#33]
```
2018-09-14T15:31:49.377Z [18609] - verbose: [plugins/fireHook] filter:middleware.render
2018-09-14T15:31:49.381Z [18609] - verbose: [plugins/fireHook] filter:widget.render:activeusers
2018-09-14T15:31:49.382Z [18609] - verbose: [plugins/fireHook] filter:widget.render:populartags
2018-09-14T15:31:49.382Z [18609] - verbose: [plugins/fireHook] filter:widget.render:forumstats
2018-09-14T15:31:49.385Z [18609] - verbose: [plugins/fireHook] filter:widget.render:recentCards
2018-09-14T15:31:49.385Z [18609] - error:  TypeError: topics.getPopularTopics is not a function
    at Object.plugin.getCategories (/home/alaa/Nodebb/forum/node_modules/nodebb-plugin-recent-cards/library.js:153:10)
    at Object.plugin.renderWidget [as method] (/home/alaa/Nodebb/forum/node_modules/nodebb-plugin-recent-cards/library.js:65:9)
    at /home/alaa/Nodebb/forum/src/plugins/hooks.js:121:12
    at /home/alaa/Nodebb/forum/node_modules/async/dist/async.js:2516:9
    at replenish (/home/alaa/Nodebb/forum/node_modules/async/dist/async.js:1011:17)
    at /home/alaa/Nodebb/forum/node_modules/async/dist/async.js:1016:9
    at eachOfLimit (/home/alaa/Nodebb/forum/node_modules/async/dist/async.js:1041:24)
    at /home/alaa/Nodebb/forum/node_modules/async/dist/async.js:1046:16
    at Object.reduce (/home/alaa/Nodebb/forum/node_modules/async/dist/async.js:2515:5)
    at fireFilterHook (/home/alaa/Nodebb/forum/src/plugins/hooks.js:113:9)
    at Object.Plugins.fireHook (/home/alaa/Nodebb/forum/src/plugins/hooks.js:93:4)
    at /home/alaa/Nodebb/forum/src/widgets/index.js:77:12
    at nextTask (/home/alaa/Nodebb/forum/node_modules/async/dist/async.js:5324:14)
    at next (/home/alaa/Nodebb/forum/node_modules/async/dist/async.js:5331:9)
    at /home/alaa/Nodebb/forum/node_modules/async/dist/async.js:969:16
    at async.waterfall.userLang (/home/alaa/Nodebb/forum/src/widgets/index.js:71:5)
2018-09-14T15:31:49.387Z [18609] - info: [app] Shutdown (SIGTERM/SIGINT) Initialised.
2018-09-14T15:31:49.388Z [18609] - info: [app] Web server closed to connections.
2018-09-14T15:31:49.390Z [18609] - info: [app] Database connection closed.
2018-09-14T15:31:49.390Z [18609] - info: [app] Shutdown complete.
[cluster] Child Process (18609) has exited (code: 1, signal: null)
```

**Comment by spwolf** [comment]
same here, had to disable widget essentials for nodebb to start with recent cards.

**Comment by sohrabafard** [comment]
in library.js
```
//topics.getPopularTopics(plugin.settings.get('popularTerm'), uid, 0, 19, renderCards);
```
fix issue!

**Comment by barisusakli** [comment]
Should be fixed in 1.0.1
